docs/01-overview/main-areas/application-connectivity/ac-04-security.md

```diff
@@ -13,6 +13,8 @@ You can [disable the TLS certificate verification](../../../03-tutorials/00-appl
 
 ## API security type
 
+<!-- TODO: is this section to be removed altogether or replaced with an alternative way that allows for API registration? -->
```

Comment:
I believe we should move it to the Application Gateway section as a list of supported auth types for the Application CRs (instead of "registered Applications"). We can move it and get rid of Application Registry references.

Reply:
Please check the changes and confirm if that's what you meant (moving just this section) or let me know if you meant to remove the whole Security doc.